ci/cloudbuild.yaml

```diff
@@ -1,84 +1,105 @@
-  args:
-  - 'image'
-  - '--output=trivy-out.table'
-  - '--no-progress'
-  - 'gcr.io/cre-tools/ts-bridge:git-$SHORT_SHA'
+  - name: "gcr.io/cloud-builders/docker"
+    id: "Build image with custom tags"
+    entrypoint: "/bin/bash"
+    args:
+      - "-c"
+      - |
+        docker buildx build --platform $_DOCKER_BUILDX_PLATFORMS \
+          -t gcr.io/$PROJECT_ID/ts-bridge:git-$SHORT_SHA \
+          -t gcr.io/$PROJECT_ID/ts-bridge:build-$BUILD_ID \
+          -t gcr.io/$PROJECT_ID/ts-bridge:$$(date -u +%Y%m%dT%H%M) \
+          -t gcr.io/$PROJECT_ID/ts-bridge:$$(cat _release_tag) \
+          -t gcr.io/$PROJECT_ID/ts-bridge:$$(cat _MAJOR_TAG) \
+          -t gcr.io/$PROJECT_ID/ts-bridge:$$(cat _MINOR_TAG) \
+          -t gcr.io/$PROJECT_ID/ts-bridge:latest \
+          --cache-from gcr.io/$PROJECT_ID/ts-bridge:latest \
+          --push .
 
-- name: gcr.io/cloud-builders/gcloud
-  entrypoint: 'bash'
-  args: [ '-c', "gcloud secrets versions access latest --secret=Ts-bridge-bot-token --format='get(payload.data)' | tr '_-' '/+' | base64 -d > git_token.txt" ]
+  - name: "docker.io/aquasec/trivy:latest"
+    id: "Scan newly built image using Trivy and create trivy-out.json"
+    args:
+      - "image"
+      - "--format=json"
+      - "--output=trivy-out.json"
+      - "--no-progress"
+      - "gcr.io/cre-tools/ts-bridge:git-$SHORT_SHA"
 
-- name: 'python:3'
-  id: 'Parse results and check for vulnerabilities'
-  entrypoint: 'bash'
-  args:
-  - '-c'
-  - |
-    pip install PyGithub
-    pip install absl-py
-    python3 ci/parse_trivy_results.py \
-      --build_id=-$BUILD_ID \
-      --commit_id=$SHORT_SHA \
-      --release_tag=$$(cat _release_tag) \
-      --repo_name=$_REPO_NAME \
-      --token_file=git_token.txt \
-      --trivy_file=trivy-out
+  - name: "docker.io/aquasec/trivy:latest"
+    id: "Scan newly built image using Trivy and create trivy-out.table"
+    args:
+      - "image"
+      - "--output=trivy-out.table"
+      - "--no-progress"
+      - "gcr.io/cre-tools/ts-bridge:git-$SHORT_SHA"
+
+  - name: gcr.io/cloud-builders/gcloud
+    entrypoint: "bash"
+    args:
+      [
+        "-c",
+        "gcloud secrets versions access latest --secret=Ts-bridge-bot-token --format='get(payload.data)' | tr '_-' '/+' | base64 -d > git_token.txt",
+      ]
+
+  - name: "python:3"
+    id: "Parse results and check for vulnerabilities"
+    entrypoint: "bash"
+    args:
+      - "-c"
+      - |
+        pip install PyGithub
+        pip install absl-py
+        python3 ci/parse_trivy_results.py \
+          --build_id=-$BUILD_ID \
+          --commit_id=$SHORT_SHA \
+          --release_tag=$$(cat _release_tag) \
+          --repo_name=$_REPO_NAME \
+          --token_file=git_token.txt \
+          --trivy_file=trivy-out
 substitutions:
   _REPO_NAME: google/ts-bridge
+  _DOCKER_BUILDX_PLATFORMS: "linux/arm64,linux/amd64,linux/arm/v7"
 options:
-  machineType: 'N1_HIGHCPU_8'
+  machineType: "N1_HIGHCPU_8"
+  env:
```

Comment:
Can you add a comment on why this is needed and whether we're planning to remove it in the future?

Reply:
Done :)